File: src/test/java/org/interview/jobtask/shortener/IdentityGeneratorTest.java
package org.interview.jobtask.shortener;

import org.junit.Test;

import static org.junit.Assert.*;

public class IdentityGeneratorTest {

    private IdentityGenerator identityGenerator = IdentityGenerator.of(Application.ALPHABET);

    @Test
    public void generate() {
        String key = identityGenerator.generate();
        assertNotNull(key);
        assertEquals(5, key.length());
        assertTrue(key.matches("^[a-zA-Z0-9]*$"));
    }
}

File: src/test/kotlin/org/interview/jobtask/shortener/storage/identity/kt/IdentityGeneratorTest.kt
package org.interview.jobtask.shortener.storage.identity.kt

import org.interview.jobtask.shortener.org.interview.jobtask.shortener.storage.identity.kt.IdentityGenerator
import org.junit.Test
import kotlin.test.assertEquals
import kotlin.test.assertTrue

class IdentityGeneratorTest {

    private val generator: IdentityGenerator = IdentityGenerator.of("123456789")

    @Test
    fun generate() {
        val key = generator.generate()
        assertEquals(5, key.length)
        assertTrue( key.matches("[0-9]*".toRegex()) )
    }

}